store/storetest/status_store.go

```diff
@@ -64,15 +64,14 @@ func testStatusStore(t *testing.T, ss store.Store) {
 		}
 	}
 
-	if err := (<-ss.Status().ResetAll()).Err; err != nil {
+	if err := ss.Status().ResetAll(); err != nil {
```

Comment:
You are making here an unrelated change, It shouldn't be changed because is not related to the PR, besides is not correct, because now we are checking that the `store.StoreResult` returned there is not a `nil` instead of checking the `err`.